docs/generic-skills.md

```diff
@@ -173,47 +91,81 @@ ledger_apis:
     gas_price: 50
 ```
 
-### Update the skill configs
+### Update the seller AEA skill configs
 
-In the generic seller skill config (`my_seller_aea/skills/thermometer/skill.yaml`) under strategy, amend the `currency_id`, `ledger_id` as follows and 
-`data_for_sale` with your own data.
-```yaml
-shared_classes:
-  strategy:
-    class_name: Strategy
-    args:
-      total_price: 10
-      seller_tx_fee: 0
-      currency_id: 'ETH'
-      ledger_id: 'ethereum'
-      is_ledger_tx: True
-      has_data_source: False
-      data_for_sale:
-        foo: {'bar': 'this is test'}
-        test: 20
-```
+In `my_seller_aea/vendor/fetchai/generi_seller/skill.yaml`, replace the `data_for_sale`, `datamodel`, and `scheme` with your data:
+```bash
+|----------------------------------------------------------------------|
+|         FETCHAI                   |           ETHEREUM               |
+|-----------------------------------|----------------------------------|
+|shared_classes:                    |shared_classes:                   |              
+|  strategy:                        |  strategy:                       |
+|     class_name: Strategy          |     class_name: Strategy         |
+|    args:                          |    args:                         |
+|      total_price: 10              |      total_price: 10             |
+|      seller_tx_fee: 0             |      seller_tx_fee: 0            |
+|      currency_id: 'FET'           |      currency_id: 'ETH'          |
+|      ledger_id: 'fetchai'         |      ledger_id: 'ethereum'       |
+|      is_ledger_tx: True           |      is_ledger_tx: True          |
+|      has_data_source: False       |      has_data_source: False      |
+|      data_for_sale:               |      data_for_sale:              |
+|        wind: 10                   |        wind: 10                  |
+|        pressure: 20               |        pressure: 20              |
+|        temperature: 26            |        temperature: 26           |
+|      datamodel:                   |      datamodel:                  |
+|        Attribute1:                |        Attribute1:               |
+|          name: country            |          name: country           |
+|          type: str                |          type: str               |
+|          is_required: True        |          is_required: True       |
+|        Attribute2:                |        Attribute2:               |
+|          name: city               |          name: city              |
+|          type: str                |          type: str               |
+|          is_required: True        |          is_required: True       |
+|      scheme:                      |      scheme:                     |
```

Comment:
this isn't the scheme, it is the specific instantiation. So maybe call it `search_data`